tests/testTransform.py

```diff
@@ -53,11 +53,13 @@
 import lsst.afw.coord as afwCoord
 import lsst.afw.table as afwTable
 import lsst.meas.base as measBase
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
```

Comment:
(with apologies for "self-reviewing"), both of those new imports are unnecessary.